msg/float64Opt.msg

```diff
@@ -0,0 +1,2 @@
+float64 value

```

Comment:
Think this idea is actually rly cool, but a bit unnecessary and we could just leave at 0